src/internal/m365/collection/drive/delta_tree.go

```diff
@@ -0,0 +1,80 @@
+package drive
+
+import (
+	"time"
+
+	"github.com/alcionai/corso/src/pkg/path"
+)
+
+// folderyMcFolderFace owns our delta processing tree.
+type folderyMcFolderFace struct {
```

Comment:
I'm not sure if I should be happy or sad about winning by default (seems like other's didn't vote) 😂

Reply:
Well that's their fault for not reading the proposal.

Reply:
Hope this name is temporary!